types/got/index.d.ts

```diff
@@ -202,7 +202,7 @@ declare namespace got {
         baseUrl?: string;
         cookieJar?: CookieJar;
         encoding?: E;
-        query?: string | object;
+        query?: string | Record<string, string>;
```

Comment:
This is breaking change. Now, user can specify object with non-string value (number or array). See https://github.com/sindresorhus/got/issues/638. So, in my opinion, right type is `Record<string, any>`. 

I think, `URLSearchParams` should be also specified here https://github.com/sindresorhus/got/tree/v9.6.0#query